ie-solver/ex1.cpp

```diff
@@ -103,34 +149,34 @@ void boundary_integral_solve(int N, double id_tol, void (*make_shape)
 	}
 
 	//	STEP 4 - LINEAR SOLVE AND ERROR CHECK
- 	ie_Mat phi(dofs, 1);
+ 	ie_Mat phi(dim*dofs, 1);
  	clock.tic();
 	skelfac.Solve(K, quadtree, phi, f);
 	clock.toc("Solve");
 	if(!timing){
-		result_f = ie_Mat(dofs,1);
+		result_f = ie_Mat(dim*dofs,1);
 		Matmul::ie_gemv(NORMAL_, 1., K_copy, phi, 0., result_f);
 		result_f -= f;
 		double lser = vec_norm(result_f)/vec_norm(f);
 		printf("Solve Error: %.10f\n", lser);
 
 	//	STEP 5 - BIE SOLVE AND OUTPUT
-		ie_Mat domain(TEST_SIZE*TEST_SIZE, 1);
+		ie_Mat domain(dim*TEST_SIZE*TEST_SIZE, 1);
 		Matmul::ie_gemv(NORMAL_, 1., K_domain, phi, 0., domain);
 
 		std::ofstream output;
-		output.open("laplace.txt");
+		output.open("ie-solver-output.txt");
```

Comment:
include a comment that this should depend on an input which selects the type of pde